invenio_rdm_records/ext.py

```diff
@@ -137,3 +137,10 @@ def init_resource(self, app):
             service=self.subjects_service,
             config=subject_record_type.resource_config_cls,
         )
+
+        #PID
+        self.pid_provider_resource = RDMPidProviderResource(
```

Comment:
minor: naming, I think everywhere else we are putting `PID` in capitals. Is another acronym like RDM.